test/evergreen.yml

```diff
@@ -5921,9 +5944,32 @@ buildvariants:
   display_name: "macOS 11.00 ARM64"
   run_on:
     - macos-1100-arm64
-  batchtime: 40320 # 28 days
+  batchtime: 120 # 2 hours
+  expansions:
+    # The cmake toolchain file is set to the mongodb toolchain gcc by defualt.
+    # Remove that configuration here and let MacOS use the default Xcode toolchain instead.
+    CMAKE_TOOLCHAIN_FILE:
+    # On Evergreen M1 Macs, the default Python is 3.9.
+    # However, we need at least Python 3.11 to avoid segfault issues with SWIG related code in the Python unit tests.
+    python_binary: '/opt/homebrew/Frameworks/Python.framework/Versions/3.11/bin/python3.11'
```

Comment:
It looks like `python_binary` is the only expansion that has different values between the 2 x macos-1100 builders. Is there a chance we could line them up? If yes, then I'd suggest using the yaml template (see [one example](https://github.com/mongodb/mongo/blob/6d077630145b3b9f1618710b575ad3cbd94386a2/etc/evergreen.yml#L302)) to save the duplicated builder config lines.

Reply:
I've created a new build variant template that factors out the common code. The Intel and ARM Macs build variants use that with the expansions that are different.